FHIR/sushi
# Update dependencies

**Description:**
Packages that remain out of date are explained in dependency notes.

**Testing Instructions:**
Confirm that all tests pass locally, and that linting works as expected. Confirm that `npm audit` reports no vulnerabilities. Confirm that `npm outdated` reports only the packages that are listed in `DEPENDENCY-NOTES.md`. Run a regression and confirm that `diff` behaves as expected, since that's one of the updated packages.

**Related Issue:**
N/A
